samples_project/Assets/SampleViewer/Samples/FeatureLayer/readme.md

```diff
@@ -4,27 +4,44 @@ Query objects from a feature layer and show it on the map.
 
 ![Feature layer](FeatureLayer.png)
 
-## How to use the sample
+## How to Setup (Feature Layer Level)
```

Comment:
I'm converting it to small case as other titles in the readme are all small cases

```suggestion
## How to set up (Feature Layer Level)
```